k9mail/src/main/res/values/colors.xml

```diff
@@ -9,6 +9,7 @@
     <color name="openpgp_green">#669900</color>
     <color name="openpgp_blue">#336699</color>
     <color name="openpgp_grey">#bbb</color>
+    <color name="openpgp_black">#000</color>
```

Comment:
Probably not. I'll do dark theme testing when everything else is done, no point switching themes all the time during development to keep up with this